codespell_lib/data/dictionary.txt

```diff
@@ -21400,24 +21542,73 @@ phylosophical->philosophical
 physcial->physical
 physial->physical
 physicaly->physically
+physican->physician
+physicans->physicians
+physicion->physician
+physicions->physicians
+physisan->physician
+physisans->physicians
+physisian->physician
+physisians->physicians
+physision->physician
+physisions->physicians
 physisist->physicist
 phython->python
 phyton->python
 phy_interace->phy_interface
 piblisher->publisher
 pich->pitch
+picknic->picnic
+pickniced->picnicked
+picknicer->picnicker
+picknicing->picnicking
+picknick->picnic
```

Comment:
picknick is an accepted variant spelling according to https://www.merriam-webster.com/dictionary/picknick

I'm not sure what we prefer to do in such cases.